EpsilonDriverControls/Inc/DriverControls.h

```diff
@@ -32,7 +32,7 @@
 
 #define MAX_FORWARD_RPM 1000
 #define MAX_REVERSE_RPM -200
-#define NON_ZERO_THRESHOLD 0.05f
+#define NON_ZERO_THRESHOLD 0.07f
```

Comment:
how come?

Reply:
When I was using the knob, the lowest value alternated between 0.05 & 0.06, so I just decided to do 0.07 to be safe.